src/spaceObjects/playerSpaceship.cpp

```diff
@@ -1042,6 +1043,10 @@ void PlayerSpaceship::onReceiveClientCommand(int32_t client_id, sf::Packet& pack
     case CMD_TARGET_ROTATION:
         packet >> target_rotation;
```

Comment:
I think it would be good to set the turnSpeed to zero here. Meaning you can get back control by normal means if a joystick control was disconnected.